### ISSUE TITLE
Make Goodies more visible

### ISSUE THREAD

**Issue by ktosiek** [samuelclay/NewsBlur#1004]
It would be great if goodies for current platform were more exposed, or at least documented in some googleable place.

Today I was looking for a way to integrate Newsblur with Firefox RSS subscription feature, and the way I've found it was:

1. Googling "firefox rss subscribe newsblur"
2. Finding the FF docs for registering RSS readers
3. Searching for "registerContentHandler" in the debuger
4. Finding there's a thing called "goodies" with integrations, apps, etc.

obviously could be better :-)

**Comment by samuelclay** [comment]
Yes, I agree, I would love to have an Apps/Goodies page. I need to get this built some day.
